doc/source/cluster/kubernetes/user-guides/ray-k8s-setup.md

```diff
@@ -0,0 +1,81 @@
+(ray-kubernetes-setup)=
+
+# Set up a Ray + Kubernetes cluster
```

Comment:
The title appears to be somewhat misleading. Although the document primarily focuses on storage, the main topic is not clearly conveyed through the current title.

Reply:
Changed title to "Best practices for deployment and storage with KubeRay".